src/types/view/index.ts

```diff
@@ -80,7 +81,13 @@ export interface ViewOutput {
   blocks: StringIndexed; // TODO: should this just be any?
   close: PlainTextElementOutput | null;
   submit: PlainTextElementOutput | null;
-  state: object; // TODO: this should probably be expanded in the future
+  state: {
+    values: {
+      [blockId: string]: {
+        [actionId: string]: any; // TODO: a union of all the input elements' output payload
```

Comment:
Yeah, this any type can be improved to support all the possible types like this https://github.com/slackapi/java-slack-sdk/blob/6d3e0ef6cb2ef109c9da01382b525187690fa80c/slack-api-model/src/main/java/com/slack/api/model/view/ViewState.java#L21 but I’m fine to go with any at this moment.